extensions/wikia/CreateNewWiki/js/WikiBuilder.js

```diff
@@ -85,8 +91,11 @@ define('ext.createNewWiki.builder', ['ext.createNewWiki.helper', 'wikia.tracker'
 		wikiDomainLabel = $nameWikiWrapper.find('label[for=wiki-domain]');
 		wikiDomainError = $nameWikiWrapper.find('.wiki-domain-error');
 		wikiDomainCountry = $nameWikiWrapper.find('.domain-country');
-		wikiLanguage = $nameWikiWrapper.find('select[name=wiki-language]');
-		wikiVertical = $descWikiWrapper.find('select[name=wiki-vertical]');
+		wikiLanguage = $nameWikiWrapper.find('input[name=wiki-language]');
+		wikiLanguageList = $nameWikiWrapper.find('.wiki-language-dropdown');
+		wikiVertical = $descWikiWrapper.find('input[name=wiki-vertical]');
+		wikiVerticalList = $descWikiWrapper.find('.wiki-vertical-dropdown');
+		wikiVerticalError = $descWikiWrapper.find('.wiki-vertical-error');
 		wikiAllAges = $descWikiWrapper.find('input[name=all-ages]');
 		allAgesDiv = $('#all-ages-div');
 		descWikiSubmitError = $descWikiWrapper.find('.submit-error');
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "descWikiSubmitError" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)